### ISSUE TITLE
Numlock sometimes gets toggled on/off with layer switch

### ISSUE THREAD

**Issue by Zetaphor** [UltimateHackingKeyboard/firmware#1014]
I have the numpad keys setup on the fn3 layer. I keep noticing that sometimes when I go into this layer numlock has been toggled off. For some reason toggling this layer is _sometimes_ toggling numlock. I can see this happening if I just keep spamming the layer activation

**Comment by Zetaphor** [comment]
Using the dongle and the bridge cable.

Screenshot of base layer:
![image](https://github.com/user-attachments/assets/e27531af-810a-46c3-9dd2-5c70ff00bad2)

Screenshot of FN3 layer:

![image](https://github.com/user-attachments/assets/1dd70f3c-8d1a-4566-96d9-bc39de7c641c)


Video of it happening:

https://share.zetaphor.com/u/Dl2bqp.mp4

**Comment by mhantsch** [comment]
Yeah, I think I had that happen too. Suddenly noticed my scroll lock was on. At that time I thought I had inadvertantly hit a key combination that resolved to NumLock; but likely that was not the case but instead I had encountered this bug too.

**Comment by kareltucek** [comment]
I assume this is no longer happening.

If you are still experiencing it, please post a UserConfig and let us know on which firmware version this is.
